lib/libzfs/libzfs_sendrecv.c

```diff
@@ -867,7 +867,7 @@ send_iterate_fs(zfs_handle_t *zhp, void *arg)
 
 	tosnap_txg = get_snap_txg(zhp->zfs_hdl, zhp->zfs_name, sd->tosnap);
 	if (tosnap_txg != 0)
-		sd->tosnap_txg = tosnap_txg;
+		sd->tosnap_txg = tosnap_txg+1;
```

Comment:
Is this change related in any way to the original issue?

Reply:
Sorry. My coworker thought he would be funny and added that when I walked away for a second. Thanks for catching it.